### PR TITLE
Update mongoose from 5.9.24 to 5.13.9 for mpath

### PR DESCRIPTION
**Pull request by visargsoneji**

### CODE REVIEW

File: broker/yarn.lock [hunk 1]
```diff
@@ -9956,31 +9966,20 @@ fsevents@^1.0.0:
...
-    bson: ^1.1.4
-    kareem: 2.3.1
-    mongodb: 3.5.9
-    mongoose-legacy-pluralize: 1.0.2
-    mpath: 0.7.0
-    mquery: 3.2.2
+    bson: ^4.2.2
+    kareem: 2.3.2
+    mongodb: 4.1.1
+    mpath: 0.8.4
```
Comment (visargsoneji) [inline broker/yarn.lock:9976]: Here, `mpath` gets updated to 0.8.4 from 0.7.0 as it is a direct dependency of `mongoose`

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n+n+n is succeeded.

**Review by anoopjb — APPROVED** [review]
LGTM

File: broker/yarn.lock [hunk 2]
```diff
@@ -9965,22 +9997,25 @@ fsevents@^1.0.0:
...
+    "@types/mongodb": ^3.5.27
     bson: ^1.1.4
-    kareem: 2.3.1
-    mongodb: 3.5.9
+    kareem: 2.3.2
+    mongodb: 3.6.11
     mongoose-legacy-pluralize: 1.0.2
-    mpath: 0.7.0
-    mquery: 3.2.2
+    mpath: 0.8.4
```
Comment (visargsoneji) [inline broker/yarn.lock:10010]: Mpath update

**Pull request merged**